src/common/auth/actions.js

```diff
@@ -35,6 +36,7 @@ export const signInFail = (error: Error): Action => ({
 
 export const signOut = () => ({ firebaseAuth }: Deps): Action => {
   firebaseAuth().signOut();
+  cookie.remove('locale');
```

Comment:
It is as we'd want the browser to tell us it's preferred language when the user logs out.

Example: An American using a public computer in France